src/charts/raster-chart.js

```diff
@@ -491,11 +504,7 @@ export default function rasterChart(parent, useMap, chartGroup, _mapboxgl) {
     const bounds = _chart.getDataRenderBounds()
     _chart._updateXAndYScales(bounds)
 
-    _vegaSpec = genLayeredVega(
-      _chart,
-      group,
-      lastFilteredSize(group.getCrossfilterId())
-    )
+    _vegaSpec = genLayeredVega(_chart)
```

Comment:
Seemed to be using an old function signature; `genLayeredVega` currently only takes one `chart` parameter